gemfiles/4.2.gemfile

```diff
@@ -0,0 +1,6 @@
+gem "activesupport", "~> 4.2"
+gem "activemodel", "~> 4.2"
+gem "activerecord", "~> 4.2"
+gem "sqlite3", "~> 1.3.6"
+gem "mysql2"
```

Comment:
The [ActiveRecord connection adapter says](https://github.com/rails/rails/blob/4-2-stable/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L3) had version restrictions:
```suggestion
gem "mysql2", ">= 0.3.13", "< 0.6.0"
```